### PR TITLE
gnrc_sixlowpan_frag: initial import of SRF parameters

### PR DESCRIPTION
**Pull request by miri64**
### Contribution description
This imports the protocol parameters for Selective Fragment Recovery (SFR). For the values I took some educated guesses based on my experience with previous experimentation with fragment forwarding.

The defines currently are based on [draft v7].

[draft v7]: https://tools.ietf.org/html/draft-ietf-6lo-fragment-recovery-07#section-7.1

### Testing procedure
This just introduces some defines and their documentation. So
```
make doc
```
and read ;-).

### Issues/PRs references
None

### CODE REVIEW

**Comment by miri64** [comment]
Rebased to current master and adapted to version 07 of the draft.

**Comment by benpicco** [comment]
Feel free to squash.

**Comment by miri64** [comment]
Squashed.

File: sys/include/net/gnrc/sixlowpan/config.h [hunk 1]
```diff
@@ -146,6 +146,133 @@ extern "C" {
...
+ * @brief   Default minimum value for fragment size (MinFragmentSize)
+ */
+#ifndef GNRC_SIXLOWPAN_SFR_MIN_FRAG_SIZE
+#define GNRC_SIXLOWPAN_SFR_MIN_FRAG_SIZE    (96U)
+#endif
+
+/**
+ * @brief   Default maximum value for fragment size (MaxFragmentSize)
+ *
+ * It must be lower than teh minimum MTU along the path. A large value augments
```
Comment (benpicco) [inline sys/include/net/gnrc/sixlowpan/config.h]:
```suggestion
 * It must be lower than the minimum MTU along the path. A large value augments
```

File: sys/include/net/gnrc/sixlowpan/config.h [hunk 2]
```diff
@@ -146,6 +146,133 @@ extern "C" {
...
+/**
+ * @brief   Default value of window size that the sender should start with
+ *          (OptWindowSize)
+ */
+#ifndef GNRC_SIXLOWPAN_SFR_OPT_WIN_SIZE
+#define GNRC_SIXLOWPAN_SFR_OPT_WIN_SIZE     (16U)
+#endif
+
+/**
+ * @brief   Default mimimum amount of time between transmissions in microseconds
```
Comment (benpicco) [inline sys/include/net/gnrc/sixlowpan/config.h]:
```suggestion
 * @brief   Default minimum amount of time between transmissions in microseconds
```

**Comment by miri64** [comment]
Fixed spelling errors and amended them directly.

**Pull request merged**